homeassistant/components/alarmdecoder/__init__.py

```diff
@@ -22,11 +24,6 @@
 from .const import (
     CONF_DEVICE_BAUD,
     CONF_DEVICE_PATH,
-    DATA_AD,
-    DATA_REMOVE_STOP_LISTENER,
-    DATA_REMOVE_UPDATE_LISTENER,
-    DATA_RESTART,
```

Comment:
These four constants can probably be removed from `const.py`

Reply:
Resolved in the latest update